Tuple extension

inxilpro opened the pull request:
## Description

This is an extension to allow you to initiate pairing sessions with [Tuple](https://tuple.app/)

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/21592/142454389-d9596683-9a02-44c1-a194-2352c1cb3d67.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

pernielsentikaer commented on extensions/tuple/src/index.tsx:
```diff
@@ -0,0 +1,114 @@
+				mask: ImageMask.Circle,
+			} }
+			actions={
+				<ActionPanel title={ `Start pairing with ${ user.email }` }>
+					<OpenInBrowserAction title="Initiate pairing session" url={ `tuple://drive?email=${ user.email }` } />
+					<CopyToClipboardAction title="Copy email address" content={ user.email } />
```
```suggestion
				<ActionPanel title={ `Start Pairing With ${ user.email }` }>
					<OpenInBrowserAction title="Initiate Pairing Session" url={ `tuple://drive?email=${ user.email }` } />
					<CopyToClipboardAction title="Copy Email Address" content={ user.email } />
```

pernielsentikaer reviewed (changes requested):
Hi @inxilpro 👋 

Awesome addition 💯   If Tuple is not installed, I think we have to gracefully handle this with stopping the loader and some more informational text in the toast

https://user-images.githubusercontent.com/38033107/142580765-7572ecac-bbee-485a-a1f2-87c3092241cc.mp4



Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

inxilpro commented:
@pernielsentikaer Alright, I've refactored the code a bunch and also added better error handling for when you don't have Tuple installed. Take another look!
